ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -19,6 +19,19 @@
 # This file defines fan-coil unit types. Valid types must include a fan and a coil of some type.
 # They must also include some control scheme (e.g. ZTC, DSP, etc.).
 
+FCU_RTM_DFSS2X_CHWZTC_SFVSC2X_HWZTC_STC_EFFC:
+  description: "FCU in a trench, with heating and cooling valves, and two fans."
+  is_canonical: true
+  implements:
+  - FCU
+  - RTM
+  - DFSS2X
+  - CHWZTC
+  - SFVSC2X
+  - HWZTC
+  - STC
+  - EFFC
```

Comment:
By definition, FCUs do not exhaust air - how can there be an exhaust fan on this unit?

Reply:
Apologies - should have been under discharge fan, thank you. I have adjusted the new FCU entity type